### PR TITLE
Fix/broken tp terms

### PR DESCRIPTION
**Pull request by junjou**

### CODE REVIEW

**Comment by junjou** [comment]
Sorry I cannot change indentation, I'm getting 500 errors on github or when it passes it's not registering.. Could any of you try and change it from 8 to 4 to see if it's working on from your side?

File: environment/sdm-environment.jsonld [hunk 1]
```diff
@@ -1,156 +1,158 @@
...
+		"dangerLevel": "https://smartdatamodels.org/dataModel.Environment/dangerLevel",
+		"dataDescriptor": "https://smartdatamodels.org/dataModel.Environment/dataDescriptor",
+		"dataProvider": "https://smartdatamodels.org/dataProvider",
+		"dateCreated": "https://smartdatamodels.org/dateCreated",
+		"dateIssued": "https://smartdatamodels.org/dataModel.Environment/dateIssued",
+		"dateModified": "https://smartdatamodels.org/dateModified",
+		"dateObserved": "https://smartdatamodels.org/dateObserved",
+		"dateObservedFrom": "https://smartdatamodels.org/dataModel.Environment/dateObservedFrom",
+		"dateObservedTo": "https://smartdatamodels.org/dataModel.Environment/dateObservedTo",
+		"depth": "https://smartdatamodels.org/dataModel.Environment/depth",
```
Comment (bobeal) [inline environment/sdm-environment.jsonld:63]: this file is basically a copy of the Environment data model from SDM and as such is already a bit crappy. I would suggest not adding things in it (it is anyway meant to be removed in the cherry-picking operation)

File: digital-twins/jsonld-contexts/environment.jsonld [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "@context": {
+        "depth": "https://smartdatamodels.org/dataModel.Environment/depth",
+        "co2": "https://smartdatamodels.org/dataModel.Environment/co2",
+        "pH": "https://smartdatamodels.org/dataModel.Environment/pH",
+        "salinity": "https://smartdatamodels.org/dataModel.Environment/salinity",
```
Comment (bobeal) [inline digital-twins/jsonld-contexts/environment.jsonld:6]: can't really see where these 2 are used in TP. looks more like SMBT attributes.

also I would prefer having this file outside the DT directory (this is not DT per se, more related to an use-case)

Comment (junjou) [reply digital-twins/jsonld-contexts/environment.jsonld:6]: > can't really see where these 2 are used in TP. looks more like SMBT attributes.

It's probably coming from the Catalog module's data which reuse some SMBT data samples.

> also I would prefer having this file outside the DT directory (this is not DT per se, more related to an use-case)

If we are cherry-picking attributes for TP only, it makes sense to gather all the new contexts in the same place. This DT directory was just created for TP in the first place...

Comment (junjou) [reply digital-twins/jsonld-contexts/environment.jsonld:6]: Discussed during daily: OK to extract and keep this directory for core TP attributes

**Pull request closed without merging**